"API rate limit exceeded" in GitHub Actions Workflow

CI Workflow is failing: 

https://github.com/FirebirdSQL/firebird-odbc-driver/actions/runs/24034920090/job/70092000216

```
Done /test/build 00:03:55.5227916
Task /test/build-test-databases
ERROR: 
{
  "message": "API rate limit exceeded for 172.203.212.193. (But here\u0027s the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)",
  "documentation_url": "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting"
}
At C:\Users\runneradmin\Documents\PowerShell\Modules\PSFirebird\0.5.0\Private\Get-FirebirdReleaseUrl.ps1:50 char:17
+ … $releases = Invoke-RestMethod -Uri $apiUrl -Headers $headers -Verbose …
+               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
At D:\a\firebird-odbc-driver\firebird-odbc-driver\firebird-odbc-driver.build.ps1:59 char:1
+ task build-test-databases {
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~
Build FAILED. 3 tasks, 1 errors, 0 warnings 00:03:56.9278349
Invoke-RestMethod: C:\Users\runneradmin\Documents\PowerShell\Modules\PSFirebird\0.5.0\Private\Get-FirebirdReleaseUrl.ps1:50
Line |
  50 |  … $releases = Invoke-RestMethod -Uri $apiUrl -Headers $headers -Verbose …
     |                ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     |  {   "message": "API rate limit exceeded for 172.203.212.193. (But here\u0027s the good news: Authenticated
     | requests get a higher rate limit. Check out the documentation for more details.)",   "documentation_url":
     | "https://docs.github.com/rest/overview/resources-in-the-rest-api#rate-limiting" }
Error: Process completed with exit code 1.
```

--- a/.github/workflows/build-and-test.yml
+++ b/.github/workflows/build-and-test.yml
@@ -35,6 +35,8 @@ jobs:
 
       - name: Build, install and test
         shell: pwsh
+        env:
+          API_GITHUB_ACCESS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: Invoke-Build test -Configuration Release -File ./firebird-odbc-driver.build.ps1
 
       - name: Upload driver (Windows)

